traffic_ops/app/lib/API/Job.pm

```diff
@@ -168,6 +168,10 @@ sub create_current_user_job {
 	if ( !defined($ds) ) {
 		return $self->not_found();
 	}
+	my $org_server_fqdn = UI::DeliveryService::compute_org_server_fqdn($self, $ds_id);
+	if ( !defined($org_server_fqdn) ) {
```

Comment:
Should check for empty string as well as `!defined`.

Reply:
The `compute_org_server_fqdn` function will never return an empty string, only `undef` or a valid-looking `orgServerFqdn` string built from the DS's primary origin.